google/auth/external_account.py

```diff
@@ -382,6 +397,50 @@ def with_token_uri(self, token_uri):
         kwargs.update(token_url=token_uri)
         return self.__class__(**kwargs)
 
+    def token_info_introspection(self, request):
+        """Introspection of token info.
```

Comment:
nit: This is redundant with the function name. IMO it would be useful to describe _what_ introspection means, because it is not clear without further digging.